tests/ci/reporter_tests.js

```diff
@@ -92,38 +95,96 @@ describe('test reporters', function(){
       reporter.finish()
       var output = stream.read().toString()
       assert.match(output, /<testsuite name="Testem Tests" tests="1" failures="0" timestamp="(.+)" time="([0-9]+)">/)
-      assert.match(output, /<testcase name="phantomjs it does &lt;cool&gt; &quot;cool&quot; &apos;cool&apos; stuff"\/>/)
+      assert.match(output, /<testcase classname="phantomjs" name="it does &lt;cool> &quot;cool&quot; \'cool\' stuff"/)
```

Comment:
Could we avoid the output change? Otherwise a breaking change version bump would be required.